src/compiler/transformers/ts.ts

```diff
@@ -877,11 +897,14 @@ namespace ts {
                 // the body of a class with static initializers.
                 setEmitFlags(classExpression, EmitFlags.Indented | getEmitFlags(classExpression));
                 expressions.push(startOnNewLine(createAssignment(temp, classExpression)));
+                addRange(expressions, pendingExpressions);

```

Comment:
As with `visitClassDeclaration`, add a comment to explain what the "pending expressions" are.